ep/lh/ep-lh-0009.md

```diff
@@ -0,0 +1,140 @@
+# linterhub: lhman
+
+| Metadata     |                                         |
+| ------------ |-----------------------------------------|
+| EP           | ep-lh-0009                              |
+| Version      | 1                                       |
+| Title        | linterhub: lhman                        |
+| Authors      | binore                                  |
+| Status       | Draft                                   |
+
+## Abstract
+
+This document describes the `lhman` - tool that allows to request engine's catalog, install engines and prepare necessary environment.
+
+## Motivation
+
+linterhub is a catalog that contains the information necessary to install, run and process the result of the work of the `linter`. To start working, you need a utility that allows you to centrally install the linter from the existing catalog.
+
+## Proposal
+
+lhman is a utility that allows you to obtain the catalog of linters and install them. For the implementation of the functional lhman is divided into `strategies`.
+
+## Strategies
+
+lhman implements the following strategies:
+
+| Name         | Description |
+| -            | -           |
+| help         | Show help   |
```

Comment:
I suggest having the same order as in sections below + order by importance or logically. My suggestion is: catalog, install, version, help.